content/prerequisites/cloud9/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Use Cloud9 IDE"
+weight: 200
+---
+
+## What is Cloud9
+
+[AWS Cloud9](https://aws.amazon.com/cloud9/) is a cloud-based IDE that lets you write, run, and debug your code with just a browser. It includes a code editor, debugger, and terminal. Cloud9 comes prepackaged with essential tools for running this workshop. Since your Cloud9 IDE is cloud-based, you can do labs from your office, home, or anywhere using an internet-connected machine.
+
+We recommend using it to run this workshop because it comes with the necessary set of tools pre-installed. If you prefer to work locally, the next section includes details on how to setup your local development environment.
```

Comment:
I think the should reword little bit. Something like, "For the best experience with the workshop and minimal setup effort, we recommend..."